app/views/equipment_models/_pending_reservations_table.erb

```diff
@@ -0,0 +1,20 @@
+<table id="table_not_yet_checked_out" class="datatable table table-striped table-bordered">
```

Comment:
This table's ID should probably be `pending_reservations_table`, this is probably a holdover from an earlier version.